blockstore/badger/blockstore.go

```diff
@@ -131,6 +131,15 @@ func (b *Blockstore) Close() error {
 	return b.DB.Close()
 }
 
+// GC runs garbage collection on the value log
+func (b *Blockstore) GC() error {
+	if atomic.LoadInt64(&b.state) != stateOpen {
+		return ErrBlockstoreClosed
+	}
+
+	return b.DB.RunValueLogGC(0.125)
```

Comment:
We need to ignore the `badger.ErrNoRewrite` error (see https://github.com/ipfs/go-ds-badger/blob/master/datastore.go#L420)

Reply:
good point, will do.

Reply:
done.